tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
Would you recommend adding an explicit `pip install tox` to `make install`, or, just telling the dev that they need to install tox? I don't mind changing `pip install tests_require` to `pip install tox` and letting tox handle test deps actually... it just still looks a bit foreign to me. In general, I think you are right that we just let tox handle it.